src/utils/browserFetcher.ts

```diff
@@ -52,8 +50,8 @@ export async function downloadBrowserWithProgressBar(registry: Registry, browser
     progressBar.tick(delta);
   }
 
-  const url = registry.downloadURL(browserName);
-  const zipPath = path.join(os.tmpdir(), `playwright-download-${browserName}-${hostPlatform}-${registry.revision(browserName)}.zip`);
+  const url = downloadURL;
+  const zipPath = path.join(os.tmpdir(), `${downloadFileName}.zip`);
```

Comment:
Is there a reason downloadFileName doesn't include `.zip` extension?

Reply:
Not really!